pkg/metadata/inventories/payload.go

```diff
@@ -40,5 +41,53 @@ func (p *Payload) MarshalJSON() ([]byte, error) {
 
 // SplitPayload breaks the payload into times number of pieces
 func (p *Payload) SplitPayload(times int) ([]marshaler.AbstractMarshaler, error) {
```

Comment:
It looks like `times` isn't used here.  I'm guessing this is to implement AbstractMarshaler?  Go doesn't make that easy to find, so I try to include a formulaic comment like this:

```suggestion
// SplitPayload implements marshaler.AbstractMarshaler#SplitPayload.
//
// In this case, the payload can only be split at the top level, so `times` is ignored
// and each top-level component is returned as a distinct payload.
func (p *Payload) SplitPayload(times int) ([]marshaler.AbstractMarshaler, error) {
```